pkg/common/bytes/buf_test.go

```diff
@@ -0,0 +1,183 @@
+package bytes
+
+import (
+	"github.com/stretchr/testify/assert"
+	"math"
+	"testing"
+)
+
+func TestBytes(t *testing.T) {
+	bytes := []byte{2, 3, 4, 5, 6}
+	byteBuffer := NewByteBuffer(bytes)
+	assert.Equal(t, byteBuffer.Bytes(), bytes)
+}
+
+func TestRead(t *testing.T) {
+	bytes := []byte{2, 3, 4, 5, 6}
+	byteBuffer := NewByteBuffer(bytes)
+	newBytes := make([]byte, len(bytes))
+	_, _ = byteBuffer.Read(newBytes)
+	assert.Equal(t, bytes, newBytes)
+}
+
+func TestReadByte(t *testing.T) {
+	bytes := []byte{2, 3, 4, 5, 6}
+	byteBuffer := NewByteBuffer(bytes)
+	for _, byteItem := range bytes {
+		readByte, _ := byteBuffer.ReadByte()
+		assert.Equal(t, byteItem, readByte)
+	}
+}
+
+func TestReadInt64(t *testing.T) {
+	byteBuffer := NewByteBuffer([]byte{127, 255, 255, 255, 255, 255, 255, 255})
+	readInt64, _ := byteBuffer.ReadInt64()
+	assert.Equal(t, readInt64, int64(math.MaxInt64))
+}
+
+func TestReadUint16(t *testing.T) {
+	byteBuffer := NewByteBuffer([]byte{255, 255})
+	readInt16, _ := byteBuffer.ReadUint16()
+	assert.Equal(t, readInt16, uint16(math.MaxUint16))
+}
+
+func TestReadUint32(t *testing.T) {
+	byteBuffer := NewByteBuffer([]byte{255, 255, 255, 255})
+	readInt32, _ := byteBuffer.ReadUint32()
+	assert.Equal(t, readInt32, uint32(math.MaxUint32))
+}
+
+func TestReadUint64(t *testing.T) {
+	byteBuffer := NewByteBuffer([]byte{255, 255, 255, 255, 255, 255, 255, 255})
+	readUint64, _ := byteBuffer.ReadUint64()
+	assert.Equal(t, readUint64, uint64(math.MaxUint64))
+}
+
+func TestWrite(t *testing.T) {
+	byteBuffer := NewByteBuffer([]byte{})
+	_, err := byteBuffer.Write([]byte{255, 255})
+	if err != nil {
+		t.Error()
+	}
+	readUint16, _ := byteBuffer.ReadUint16()
+	assert.Equal(t, readUint16, uint16(math.MaxUint16))
+}
+
+func TestWriteString(t *testing.T) {
+	byteBuffer := NewByteBuffer([]byte{})
+	_, err := byteBuffer.WriteString("seata")
+	if err != nil {
+		t.Error()
+		return
+	}
+	seataBytes := []byte{115, 101, 97, 116, 97}
```

Comment:
Maybe it would be more concise to use t.Run() to run the test cases based on the test package?

for details, see:

- [add rm unit test](https://github.com/seata/seata-go/pull/192/files#diff-771a622764b6837383505f26e17ed079f8e1c3b1a136b49ac8dd9b533c44117a)